gobblin-data-management/src/main/java/gobblin/data/management/copy/IdentityCopyableDatasetFinder.java

```diff
@@ -0,0 +1,32 @@
+package gobblin.data.management.copy;
```

Comment:
This class is unused at the moment, and it is replaceable by `CopyableGlobDatasetFinder`, can we remove it?